src/main/resources/config/liquibase/changelog/20210125101100_changelog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="b-fein" id="20210125101100">
+        <addColumn tableName="programming_exercise_test_case">
+            <column name="visibility" type="VARCHAR(255)"/>
```

Comment:
can we define a default value here to prevent null?

Reply:
I set a default of ALWAYS, as only feedbacks visible to students are stored in the database for older feedbacks. Also the default visibility for test cases is ALWAYS.

Reply:
We're still missing a default Value and defaultOnNull Value here in the sql definition.

Reply:
Sorry, I somehow missed that one, as the code also sets a default of `ALWAYS`.
I didn’t find useful documentation on `defaultOnNull`. I assume `defaultOnNull="true"` is what we want here?

(Shouldn’t `defaultValue` alone be enough tell the database to replace inserts omitting `visibility` values with the default? Is the difference that this also sets a default when the insert explicitly states `visibility=null`?)